src/site/content/en/blog/fugu-status/index.md

```diff
@@ -258,6 +264,23 @@ latest version of Chrome, and in many cases other Chromium based browsers.
           <em>Updated December 18, 2019</em>
         </td>
       </tr>
+      <tr>
+        <td>
+          <a href="/shape-detection/">Shape&nbsp;Detection</a>
+        </td>
+        <td>
+          Shape detection opens up native implementations of shape
+          detection services and exposes them through a set of JavaScript
+          interfaces. Shape detection three separate APIs, one of which
+          has already shipped.<br>
+          <ul>
+            <li>Barcade Detection API (Enabled by default)</li>
+            <li><a href="https://www.chromestatus.com/feature/5678216012365824">Face Detection API</a></li>
+            <li><a href="https://www.chromestatus.com/feature/5644087665360896">Text Detection API</a></li>
+          </ul>
```

Comment:
I'd remove this.

Reply:
This will let interested parties get updates. How that works on Chrome Status is being renovated currently. I'll send you the plan if you're interested.